exs/mundo_1/java/022.java

```diff
@@ -0,0 +1,28 @@
+/*
+    ### 022 - Analisador de textos
+    Crie um programa que leia o nome completo de uma pessoa e mostre:
+    em lestras maiúsculas, minúsculas, a quantidade de letras sem espaço, e quantas letras tem o primeiro nome
```

Comment:
Correção ortográfica necessária nesse comentário

```suggestion
    em letras maiúsculas, minúsculas, a quantidade de letras sem espaço e quantas letras tem o primeiro nome
```